### DISCUSSION TITLE
大家写的时候，正文是用英文逗号还是中文逗号？哪个更美观符合规范些？

### DISCUSSION THREAD

**Discussion by giserzy** [whutug/whu-thesis#188] [General]
大家写的时候，正文是用英文逗号还是中文逗号？哪个更美观符合规范些？

**Comment by sikouhjw** [comment]
正文用中文，数学公式内用英文。

如果是数学专业，可以统一英文。

只要全文风格是统一的（指某部分用某风格，另一部分用另一风格），问题不大。

**Comment by zepinglee** [comment]
有份行业标准 CY/T 154—2017《中文出版物夹用英文的编辑标准》可以供参考。其中大致是，中文部分用中文符号，西文部分用西文符号。数学符号一般也跟西文一致。
